wafer/schedule/templates/wafer.schedule/embed_schedule.html

```diff
@@ -0,0 +1,79 @@
+{% load static from staticfiles %}
+{% load i18n %}
+<!DOCTYPE html>
+<html>
+<head>
+  <title>{% block title %}{% trans "Schedule" %} - {{ WAFER_CONFERENCE_NAME }}{% endblock %}</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="{% static 'vendor/bootstrap/dist/css/bootstrap.min.css' %}" rel="stylesheet">
+  <link href="{% static 'vendor/@fortawesome/fontawesome-free-webfonts/css/fontawesome.css' %}" rel="stylesheet">
+  <link href="{% static 'vendor/@fortawesome/fontawesome-free-webfonts/css/fa-solid.css' %}" rel="stylesheet">
+  <link href="{% static 'css/wafer.css' %}" rel="stylesheet">
+  {% block extra_head %}{% endblock %}
+</head>
+<body>
+{% block content %}
+<section class="wafer wafer-schedule">
+  <div class="wafer_schedule">
```

Comment:
If we're going to have this view, I suspect the core schedule logic should be factored out into an include or a block.

Reply:
True.